SPECS/openssl.spec

```diff
@@ -1,139 +1,72 @@
+%global package_speccommit 702826a0634052f821626edbf7764d892c5a5a81
+%global usver 3.0.9
+%global xsver 2.0.1
+%global xsrel %{xsver}%{?xscount}%{?xshash}
```

Comment:
I'm not sure we want to reintroduce these

Reply:
It will be easier to rebase later imo

Reply:
The broader question is: what's our upstream for packaging this component. 

Previously, we moved away from XenServer on openssl, because their package was missing CVE fixes, and so we rebased on CentOS. 

But now, with this OpenSSL 3 update, it's easiest to move back to XS packaging, after checking if we agree with theirs. It might be good to compare with the packaging of the SRPM they forked to make their own, and this would allow us to decide with all key data available.

I suggest we start with their packaging to have a first build available soon, but also do the rest of the analysis and if necessary base our package on another upstream that we trust with their CVE handling.

Reply:
Removed old patches, added explanations, incremented the version number